benches/lzma.rs

```diff
@@ -31,13 +31,28 @@ fn decompress_bench(compressed: &[u8], b: &mut Bencher) {
     });
 }
 
+fn decompress_stream_bench(compressed: &[u8], b: &mut Bencher) {
+    b.iter(|| {
+        let mut stream = lzma_rs::decompress::Stream::new(Vec::new());
+        stream.write_all(compressed).unwrap();
+        stream.finish().unwrap();
```

Comment:
I belive the resulting decompressed output should be the result of the lambda function passed to `Bencher::iter`, to make sure it's not optimized away. However, other benchmarks don't currently do that, but I can send a pull request to fix that accordingly.

Reply:
Good catch.